### DISCUSSION TITLE
Filter languages?

### DISCUSSION THREAD

**Discussion by StrangeMinds1960** [mastodon/mastodon#27172] [Q&A] [answered]
Under 'preferences > other', you can select languages to display in public timelines, but what are defined as 'public timelines'?  I've got my options set to just English, but still see other languages (mainly Russian & Chinese) on my federated timeline, is that not a public timeline?

**Comment by StrangeMinds1960** [comment, marked as answer]
Ok, I've discovered the moderation interface, and when I open the posts in that the language is either set as English or None, so I guess the language filter can't (or won't) do anything with it

**Comment by N1N74** [comment]
I still see unrelated languages on my homepage at mastodon.social where I can't check the language of posts but the same post on oldbytes.space which shows language on top right corner on user's page, show that it is listed as 'FR' a language I did not select. Maybe something isn't working on mastodon.social but unsure how I can report bugs as I'm not a CS student lol

funnily the user's home server is mastodon.social and so I think they're tagging the language properly
my homepage is taken over by languages I don't read lol
